config/grub/grub/local_efi.cfg

```diff
@@ -4,6 +4,8 @@ menuentry "local" --class gnu-linux --class gnu --class os {
       chainloader /efi/opensuse/grub.efi
     elif [ -f /efi/sles/shim.efi ] ; then
       chainloader /efi/sles/grub.efi
+    elif [ -f (${root})/efi/sle/grub.efi ] ; then
+      chainloader (${root})/efi/sle/grub.efi
```

Comment:
```suggestion
    elif [ -f /efi/sle/grub.efi ] ; then
      chainloader /efi/sle/grub.efi
```

Reply:
Moreover, we need to duplicate this downwards into the else branch to achieve the desired effect.